### PR TITLE
Add document of IAbpClaimsPrincipalFactory.

### PR DESCRIPTION
**Pull request by maliming**
https://github.com/abpframework/abp/issues/6089

### CODE REVIEW

**Comment by hikalkan** [comment]
This is for custom properties but we are showing an example that uses EditionId (thats a standard claim defined by ABP framework), which may confuse people. I think we should provide another example that feels that it is really custom, like `SocialSecurityNumber` for example.

**Comment by maliming** [comment]
@hikalkan Changed

File: docs/en/Authorization.md [hunk 1]
```diff
@@ -393,6 +393,39 @@ public override void ConfigureServices(ServiceConfigurationContext context)
...
+    {
+        var identity = context.ClaimsPrincipal.Identities.FirstOrDefault();
+        if (identity != null)
+        {
+            var currentUser = context.ServiceProvider.GetRequiredService<ICurrentUser>();
+            if (currentUser.Id.HasValue)
+            {
+                var userManager = context.ServiceProvider.GetRequiredService<IdentityUserManager>();
+                var user = await userManager.GetByIdAsync(currentUser.Id.Value);
+                if (user?.SocialSecurityNumber != null)
```
Comment (PMExtra) [inline docs/en/Authorization.md:415]: It's not possible. `user` is an instance of `IdentityUser` that does NOT have `SocialSecurityNumber` property.
It should be something like `user.GetProperty("SocialSecurityNumber", default(string))`.

Comment (maliming) [reply docs/en/Authorization.md:415]: It’s just a example.

**Review by hikalkan — APPROVED** [review]

**Pull request merged**